Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1637,16 +1626,12 @@ PhysicalParticleContainer::PushP (int lev, Real dt,
 
     if (do_not_push) return;
 
-    const std::array<Real,3>& dx = WarpX::CellSize(lev);
-
```

Comment:
Fixes the warning:
`./Source/Particles/PhysicalParticleContainer.cpp:1624:31`: warning: unused variable ‘dx’ [-Wunused-variable]
     const std::array<Real,3>& dx = WarpX::CellSize(lev);